apps/mobile/mobile-app/components/ui/utils/use-break-point-value.ts

```
@@ -1,103 +1,103 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState } from 'react'
+import { Dimensions, useWindowDimensions } from 'react-native'
 
-import resolveConfig from 'tailwindcss/resolveConfig';
-import * as tailwindConfig from 'tailwind.config';
+import * as tailwindConfig from 'tailwind.config'
+import resolveConfig from 'tailwindcss/resolveConfig'
 
-const TailwindTheme = resolveConfig(tailwindConfig as any);
-const screenSize = TailwindTheme.theme.screens;
+const TailwindTheme = resolveConfig(tailwindConfig as any)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)